kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/KafkaDataConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.wepay.kafka.connect.bigquery.convert;
+
+
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class KafkaDataConverter {
+
+    public static final String KAFKA_DATA_FIELD_NAME = "kafkaData";
+    public static final String KAFKA_DATA_TOPIC_FIELD_NAME = "topic";
+    public static final String KAFKA_DATA_PARTITION_FIELD_NAME = "partition";
+    public static final String KAFKA_DATA_OFFSET_FIELD_NAME = "offset";
+    public static final String KAFKA_DATA_INSERT_TIME_FIELD_NAME = "insertTime";
+
+    public static Field getKafkaDataField() {
```

Comment:
Just feel like we should put this function else where. IMO, all public function in a `Converter` class should be "Orange convertFruit(Apple)" 😛

Reply:
Changed the class name to be KafkaDataBuilder instead of KafkaDataConverter since we are not converting anything here.

Reply:
LGTM!